solarforecastarbiter/conftest.py

```diff
@@ -1279,6 +1280,15 @@ def index(category):
     return index
 
 
+@pytest.fixture
+def preprocessing_result_types():
```

Comment:
Is this used anywhere?  If not, probably ok to remove otherwise there is a missing space after Forecast/Observation

Reply:
Yes, it is. I've added it to be used in the `raw_report` fixture. Not sure why the test to compare the rendered html macros would be working with that missing space will need to check...